.github/workflows/publish_s3.yml

```diff
@@ -133,7 +133,7 @@ jobs:
 
           # USE_KMS in build_requirements.yml
           for f in secureboot.{{pk,kek,db}.{crt,der,auth},aws-efivars}; do
-            cp "cert/gardenlinux-$f" "$CNAME/$CNAME.$f"
+            cp "cert/gardenlinux-nightly-$f" "$CNAME/$CNAME.$f"
```

Comment:
This construction is currently a workaround as cached files are not accessible by subsequent workflows (which is based on the naming of the cache not really true but you can't guarantee that the cache still exists). If certificates are created in a single source of true way (in `.github/workflows/build_requirements.yml`) these files should be stored in an artifact and then we do not need these lines anymore.